src/Sulu/Bundle/AdminBundle/Resources/js/containers/Link/Link.js

```diff
@@ -161,13 +220,26 @@ class Link extends Component<Props> {
                             value={provider}
                         >
                             {allowedTypes.map((key) => (
-                                <SingleSelect.Option key={key} value={key}>{key}</SingleSelect.Option>
+                                <SingleSelect.Option key={key} value={key}>{linkTypeRegistry.getTitle(key)}</SingleSelect.Option>
                             ))}
                         </SingleSelect>
                     </div>
                     <div className={linkStyles.itemContainer}>
                         <div className={itemClass} onClick={disabled || this.handleTitleClick} role="button">
-                            {title}
+                            {this.displayPropertiesLoading && '…'}
+                            {!this.displayPropertiesLoading && value && (
+                                <div className={linkStyles.columnList}>
+                                    {displayProperties.map((displayProperty) => (
+                                        <span
+                                            className={linkStyles.itemColumn}
+                                            key={displayProperty}
+                                            style={{width: 100 / displayProperties.length + '%'}}
```

Comment:
Why this inline styling? Also calculation by string length doesn't work with our font as it is not a monofont.